level1c4pps/seviri2pps_lib.py

```diff
@@ -260,6 +260,7 @@ def set_attrs(scene):
         idtag = PPS_TAGNAMES[band]
         scene[band].attrs['id_tag'] = idtag
         scene[band].attrs['description'] = 'SEVIRI ' + str(band)
+        scene[band].attrs['wavelength'] = scene[band].attrs['wavelength'][0:3]
```

Comment:
You could use `attrs['wavelength'].min/max/central` to improve readability. In the tests you could do

```python
from satpy.dataset.dataid import WavelengthRange
scene = Scene(..., attrs={..., 'wavelength': WavelengthRange(1, 2, 3)})
```